apps/todolist-mongo-go/Dockerfile

```
@@ -1,18 +1,18 @@
 #FROM registry.access.redhat.com/ubi8/ubi-minimal
-FROM golang:1.17.8-alpine AS build-env
+FROM golang:1.23-alpine AS build-env
```

```suggestion
FROM --platform=$BUILDPLATFORM golang:1.23-alpine AS build-env
```

This should be faster to build.